espflash/README.md

```diff
@@ -1,89 +1,94 @@
 
 ## Usage
 
 ```text
-espflash 2.0.0-dev
-A command-line tool for flashing Espressif devices over serial
-
-USAGE:
-    espflash <SUBCOMMAND>
-
-OPTIONS:
-    -h, --help       Print help information
-    -V, --version    Print version information
-
-SUBCOMMANDS:
-    board-info         Display information about the connected board and exit without flashing
-    flash              Flash an application to a target device
-    help               Print this message or the help of the given subcommand(s)
-    monitor            Open the serial monitor without flashing
-    partition-table    Operations for partitions tables
-    save-image         Save the image to disk instead of flashing to device
-    write-bin          Writes a binary file to a specific address in the chip's flash
+A library and command-line tool for flashing Espressif devices
+
+Usage: espflash <COMMAND>
+
+Commands:
+  board-info       Display information about the connected board and exit without flashing
+  flash            Flash an application to a target device
+  monitor          Open the serial monitor without flashing
+  partition-table  Operations for partitions tables
+  save-image       Save the image to disk instead of flashing to device
+  write-bin        Writes a binary file to a specific address in the chip's flash
+  help             Print this message or the help of the given subcommand(s)
+
+Options:
+  -h, --help     Print help information
+  -V, --version  Print version information
 ```
 
-## Compile-time features
+### Cargo Runner
 
- - `raspberry`: enables configuring DTR and RTS GPIOs which are necessary to use a Raspberry Pi's
-   internal UART peripherals. This feature is optional (external USB <-> UART converters work
-   without it) and adds a dependency on [`rppal`](https://crates.io/crates/rppal).
-
-## Configuration
-
-You can also specify the serial port and/or expected VID/PID values by setting them in the configuration file. This file is in different locations depending on your operating system:
-
-| Operating System | Configuration Path                                                       |
-| :--------------- | :----------------------------------------------------------------------- |
-| **Linux:**       | `/home/alice/.config/espflash/espflash.toml`                             |
-| **Windows:**     | `C:\Users\Alice\AppData\Roaming\esp\espflash\espflash.toml`              |
-| **macOS:**       | `/Users/Alice/Library/Application Support/rs.esp.espflash/espflash.toml` |
-
-An example configuration file may look as follows (note that TOML does _not_ support hexadecimal literals):
+You can also use `espflash` as a Cargo runner by adding the following to your project's `.cargo/config.toml` file, for example:
 
 ```toml
-[connection]
-serial = "/dev/ttyUSB0"
-
-[[usb_device]]
-vid = "303A"
-pid = "8000"
+[target.'cfg(any(target_arch = "riscv32", target_arch = "xtensa"))']
+runner = "espflash --baud=921600 --ram --monitor /dev/ttyUSB0"
```

Comment:
the "--ram" option is maybe a bit confusing to first time users?

Reply:
I had just copy-pasted this from the previous README but I think you're right, it probably does more harm than good being there. Will update it!